MAINTAINERS.md

```diff
@@ -0,0 +1,282 @@
+
+- Those with publish access are expected to use npm's 2FA.
+- This level of access will be granted by the current owners to those maintainers who have earned the project's trust.
+
+## Mocha's Decision-Making Process
+Mocha follows a [consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) process.  In other words, all maintainers attempt to come to agreement.  If that fails, we decide by a simple vote.
+
+Active maintainers will make an effort to solicit feedback from others before making important or potentially controversial decisions.  Given the varying geographical distribution and availability of the maintenance team, we resolve to do the best we can to solicit feedback.
+
+In other words, to have your opinion heard, participate regularly.  The rest of the team won't wait on feedback that isn't necessarily forthcoming!
+
+## Communication
+
+Maintainers will mainly gather in [the mochajs/contributors channel](https://gitter.im/mochajs/contributors).  This is a *public* channel, and *anyone* can join.
+Videoconference (or audio) calls may happen on a regular or irregular basis, as schedules allow. This is mainly because we have Real Lives and time zones suck.
+
+## Labeling Issues & Pull Requests
+
+All new issues will need to be triaged, and pull requests must be examined.  Maintainers must understand [Semantic Versioning](http://semver.org) ("SemVer"), as Mocha follows it strictly.
+
+> If you see an issue or PR that could use some labels, please add them!
+
+The following list is incomplete, but it's better than nothing:
+
+### Semantic Versioning
+
+*All* issues should have one of these three labels:
+
+- `semver-patch` for bug fixes, documentation, development environment changes, CI and tests
+- `semver-minor` for enhancements ("features")
+- `semver-major` for backwards-incompatible ("breaking") changes to the *output*, *API*, or environment support
+
+Pull requests don't need `semver-*` labels, *unless* there is no associated issue.
+
+An issue or PR which will introduce a breaking change will be `semver-major`, *regardless* of any other label.
+
+### Questions
+
+Support questions should be answered if possible, but the user should be directed to the chat room, StackOverflow, or Google Groups for further help, *if* it is indeed a Mocha-related issue.  
+
+If it's *not* a Mocha problem (people tend not to believe this), you may want to show a counter-example.  It's often helpful to direct the issue author to the responsible project, if you can determine what that is.
+
+- `common-mistake`: Issues which are *repeatedly* asked support questions or misunderstandings.  This may also apply to questions which receive a lot of 👍's
+- `question`: Add this label if it's just a support question
+
+### Bugs
+
+- `unconfirmed`: A maintainer has not yet or cannot reproduce; typically `needs-feedback` follows (see "Feedback & Follow-ups" below)
+- `bug`: A confirmed bug
```

Comment:
Currently much of our usage is "bug" for both of these with the addition of either "confirmed" or "unconfirmed". Do we want to update those?

Reply:
I can change them to something like `confirmed-bug` and `unconfirmed-bug`